services/minespace-web/src/components/pages/Project/InformationRequirementsTablePage.tsx

```diff
@@ -217,8 +214,7 @@ export const InformationRequirementsTablePage = () => {
 
     await handleFetchData();
     setUploadedSuccessfully(true);
-    setEditMode(!isEditMode);
-    return cleanFilePondFile();
```

Comment:
I'm not 100% sure what this function is supposed to do, but it just errored out every time. 🤷‍♀️